### PR TITLE
Move static check to GitHub actions

### PR DESCRIPTION
**Pull request by stereomon**
Moved CodeStyle and static checks to github actions.

https://github.com/propelorm/Propel2/issues/1668

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+name: PropelORM CI
+
+on: [
+    push,
+    pull_request
```
Comment (dereuromark) [inline .github/workflows/ci.yml:5]: do we maybe only need one? pull_request?

File: README.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 Propel2 is an open-source Object-Relational Mapping (ORM) for PHP.
 
 [![Build Status](https://travis-ci.org/propelorm/Propel2.svg?branch=master)](https://travis-ci.org/propelorm/Propel2)
+[![Github actions Status](https://github.com/propelorm/Propel2/workflows/PropelORM%20CI/badge.svg?branch=master)](https://github.com/propelorm/Propel2/actions?query=workflow%3APropelORM%20CI+branch%3Amaster)
```
Comment (dereuromark) [inline README.md:6]: Can we call the workflow just CI?
As we do for others as well?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (stereomon) [reply .github/workflows/ci.yml:4]: Updated.

File: README.md (hunk 2 above)
Comment (stereomon) [reply README.md]: Updated.

**Review by dereuromark — APPROVED** [review]

**Pull request merged**